### PR TITLE
Improvement/various

### PR DESCRIPTION
**Pull request by fredericlestage-vdm**
- Improved on the phone input
- Updated names and values (oss --> software and license --> licence)
- Updated Breadcrumbs
- Optimized the form and schema solution
- Optimized the tables pages
- Standardized the yml data

### CODE REVIEW

**Comment by fredericlestage-vdm** [comment]
Fixes for #630, #637, #636, #634 & #631

File: _includes/breadcrumbs.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        <li><a href="{{ site.data.i18n.general.nav.standard.link[page.lang] }}">{{ site.data.i18n.general.nav.standard[page.lang] }}</a></li>
+      {%- endif -%}
+
+      {%- if page.ref == 'codeForm' or page.ref == 'softwareForm' or page.ref == 'standardForm' -%}
+        <li><span style="margin-left:10px">{{ site.data.i18n.general.nav.form[page.lang] }}</span></li>
+      {%- elsif page.ref == 'codeGroups' -%}
+        <li><span style="margin-left:10px">{{ site.data.i18n.general.nav.groups[page.lang] }}</span></li>
+      {%- elsif page.ref =='codeSchema' or page.ref == 'softwareSchema' or page.ref == 'standardSchema' -%}
+        <li><span style="margin-left:10px">{{ site.data.i18n.general.nav.schema[page.lang] }}</span></li>
+      {%- endif -%}
```
Comment (smellems) [inline _includes/breadcrumbs.html]: We don't need this last part of the breadcrumb since it just repeats the title of the page

Comment (fredericlestage-vdm) [reply _includes/breadcrumbs.html]: In that case, I suggest remove the breadcrumbs altogether. The site's navigation is not super complex and each page title is the same as the breadcrumb path, for instance, in "Form with validation for open source code", we got both "form" and "code". Even for top level pages (code, software, standard, doc), we already have the dark blue menu that can be used for navigating between them.

Comment (smellems) [reply _includes/breadcrumbs.html]: Before removing the breadcrumb we should check with the re-design.

But following the canada.ca WET Style Guide, the breadcrumb doesn't include the current page

Comment (fredericlestage-vdm) [reply _includes/breadcrumbs.html]: However, if we still want to keep them, I don't think the last link are unnecessary. From my personal experience and all the example I just went through, breadcrumbs usually have a node showing where you are now, even if it repeat's the page title.

Comment (fredericlestage-vdm) [reply _includes/breadcrumbs.html]: I'll remove the last node for now, and we'll see with redesign what we do next.

File: _data/i18n/general.yml [hunk 2]
```diff
@@ -62,6 +62,15 @@ nav:
...
       fr: https://www.canada.ca/fr.html
+  form:
+    en: Form
+    fr: Formulaire
+  groups:
+    en: Groups
+    fr: Groupes
+  schema:
+    en: Schema
+    fr: Schéma
```
Comment (smellems) [inline _data/i18n/general.yml]: Can we also delete this?  Thanks!

**Pull request merged**